### PR TITLE
Level 3 - adding a list

### PR DESCRIPTION
**Pull request by ampuscas**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java [hunk 1]
```diff
@@ -58,10 +65,20 @@ public boolean configure (StaplerRequest req, JSONObject formData) throws FormEx
...
         public FormValidation doCheckGlobalName(@QueryParameter String globalName) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
             if (globalName.matches("[a-zA-Z\\s]+")) {
                 return FormValidation.ok();
             }
             return FormValidation.warning("Name can only contain lowercase and uppercase letters and spaces");
         }
+
+        public ListBoxModel doFillCategoriesItems() {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```
Comment (jtnord) [inline src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java:76]: who can (or is supposed to) be able to use this?  
IIRC the `ViewsTabBar` is added to `ListViews` and those can be created and configured by regular users (so long as they have Item.Create/Item.Configure where they are to be used?

File: src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java [hunk 2]
```diff
@@ -58,10 +65,20 @@ public boolean configure (StaplerRequest req, JSONObject formData) throws FormEx
         }
 
         public FormValidation doCheckGlobalName(@QueryParameter String globalName) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```
Comment (jtnord) [inline src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java:68]: this prevents access to check the name, but anyone can call configure (there is no protection on that), is that correct?

Comment (ampuscas) [reply src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java:68]: For the `configure` method, the permissions are checked in the class `Jenkins`, method `doConfigSubmit` (`checkPermission(MANAGE);`)
And I have checked, if not logged in, I cannot access http://localhost:8080/jenkins/configure

File: src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java (hunk 1 above)
Comment (ampuscas) [reply src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java]: I've added these permissions

**Pull request merged**